packages/sirius-web/frontend/sirius-web-application/src/extension/DefaultExtensionRegistry.tsx

```diff
@@ -419,6 +419,7 @@ const defaultSettingPages: ProjectSettingTabContribution[] = [
     id: 'images',
     title: 'Images',
     icon: <ImageIcon />,
+    feature: 'imageEnabled',
```

Comment:
I only change this extension point, but the idea would be to change `ComponentExtension` and `DataExtension`